Source/WebCore/platform/gamepad/wpe/WPEGamepad.cpp

```diff
@@ -84,6 +89,15 @@ void WPEGamepad::absoluteAxisChanged(unsigned axis, double value)
     WPEGamepadProvider::singleton().scheduleInputNotification(*this, WPEGamepadProvider::ShouldMakeGamepadsVisible::Yes);
 }
 
+void WPEGamepad::analogButtonChanged(unsigned button, double value)
+{
+    m_lastUpdateTime = MonotonicTime::now();
+    m_buttonValues[button].setValue(value);
```

Comment:
can we assure that the lower layers will send values between 0 and 1? 

would it make sense to clamp or even assert those received values?